Issues building on ppc64le and i586 (gcc-go).

When trying to build on `master` (which you can see in [this OBS repo][v:c:e:containerd]), you get the following error:

```
% go-5 build '-gccgoflags=-Wl,--add-needed -Wl,--no-as-needed -static-libgo -ldl' -x -o containerd-0.1.0+git389483d github.com/docker/containerd/containerd
vendor/src/github.com/docker/containerd/supervisor/monitor_linux.go:8:2: no buildable Go source files in /home/abuild/rpmbuild/BUILD/containerd-git.389483d/vendor/src/github.com/docker/containerd/archutils
```

This doesn't happen on `0.1.0`, so it's clearly a regression. I can't figure out why it's failing, because both of the files in `archutils` should work properly.

[v:c:e:containerd]: https://build.opensuse.org/package/show/Virtualization:containers:experimental/containerd

--- a/archutils/epoll.go
+++ b/archutils/epoll.go
@@ -1,4 +1,5 @@
-//+build !arm64,linux
+// +build linux,!aarch64
+
 package archutils
 
 import (

--- a/archutils/epoll_aarch64.go
+++ b/archutils/epoll_aarch64.go
@@ -5,31 +5,31 @@ package archutils
 // #include <sys/epoll.h>
 /*
 int EpollCreate1(int flag) {
-        return epoll_create1(flag);
+	return epoll_create1(flag);
 }
 
-int EpollCtl(int efd, int op, int sfd, int Events, int Fd) {
-        struct epoll_event event;
-        event.events = Events;
-        event.data.fd = Fd;
+int EpollCtl(int efd, int op,int sfd, int events, int fd) {
+	struct epoll_event event;
+	event.events = events;
+	event.data.fd = fd;
 
-        return epoll_ctl(efd, op, sfd, &event);
+	return epoll_ctl(efd, op, sfd, &event);
 }
 
-typedef struct Event{
-        uint32_t events;
-        int fd;
+struct event_t {
+	uint32_t events;
+	int fd;
 };
 
 struct epoll_event events[128];
-int run_epoll_wait(int fd, struct Event *event) {
-        int n, i;
-        n = epoll_wait(fd, events, 128, -1);
-        for (i = 0; i < n; i++) {
-                event[i].events = events[i].events;
-                event[i].fd = events[i].data.fd;
-        }
-        return n;
+int run_epoll_wait(int fd, struct event_t *event) {
+	int n, i;
+	n = epoll_wait(fd, events, 128, -1);
+	for (i = 0; i < n; i++) {
+		event[i].events = events[i].events;
+		event[i].fd = events[i].data.fd;
+	}
+	return n;
 }
 */
 import "C"
@@ -57,8 +57,8 @@ func EpollCtl(epfd int, op int, fd int, event *syscall.EpollEvent) error {
 }
 
 func EpollWait(epfd int, events []syscall.EpollEvent, msec int) (int, error) {
-	var c_events [128]C.struct_Event
-	n := int(C.run_epoll_wait(C.int(epfd), (*C.struct_Event)(unsafe.Pointer(&c_events))))
+	var c_events [128]C.struct_event_t
+	n := int(C.run_epoll_wait(C.int(epfd), (*C.struct_event_t)(unsafe.Pointer(&c_events))))
 	if n < 0 {
 		return int(n), fmt.Errorf("Failed to wait epoll")
 	}

--- a/archutils/epoll_aarch64.go
+++ b/archutils/epoll_aarch64.go
@@ -1,4 +1,4 @@
-// +build arm64,linux
+// +build linux,aarch64
 
 package archutils
 

